dali/util/CMakeLists.txt

```diff
@@ -71,3 +81,5 @@ endif()
 set(DALI_INST_HDRS ${DALI_INST_HDRS} PARENT_SCOPE)
 set(DALI_SRCS ${DALI_SRCS} PARENT_SCOPE)
 set(DALI_TEST_SRCS ${DALI_TEST_SRCS} PARENT_SCOPE)
+
+list(FILTER DALI_SRCS EXCLUDE REGEX ".*nvml_wrap.cc")
```

Comment:
I am wondering, why do we need this?

Reply:
Because we create a separate OBJECT library from this file and we don't want to have the same file compiled in different paths, and linked multiple times.